lasagne/layers/special.py

```diff
@@ -6,12 +6,13 @@
 from ..utils import as_tuple
 from .base import Layer, MergeLayer
 
-
 __all__ = [
     "NonlinearityLayer",
     "BiasLayer",
     "InverseLayer",
     "TransformerLayer",
+    "ParametricRectifierLayer",
+    "parametric_relu"
```

Comment:
Should end with a comma. The idea is that you can always add or insert items without touching existing lines.